docs/SRS-Volere/SRS.tex

```diff
@@ -77,22 +79,44 @@ \subsection{Maintenance Users and Service Technicians}
 
 \section{Mandated Constraints}
 \subsection{Solution Constraints}
-\lips
+\textbf{Mobile Device}\\
+\underline{Description}: User will interact with the system through an app on their mobile device (smartphone).\\
+\underline{Rational}: Mobile devices are widly used and ubiquitous. While at a grocery store, users may need access to the system.\\
+\underline{Fit Criterion}: User interface will run entirely on a mobile device.\\
+\\
+\textbf{Internet access}\\ %%todo: is this too solution specific?
+\underline{Description}: User must have internet access.\\
+\underline{Rational}: System servers and databases are used to store collective user data and process that data. User's devices are required to communicate to with the servers and databases via internet access.\\
+\underline{Fit Criterion}: User interface will have internet access.\\
+\\
+\textbf{Device with Camera}\\
+\underline{Description}: User must be equipped with a device that has a camera. This should be quite common as most smartphones have a camera.\\
+\underline{Rational}: Users will be using the system while they are out on their grocery trips. Thus, the system needs to have a quick and easy method of data input. This also makes it easier for those who have their hands full or do not have the finger mobility to effectively type on their phone keyboards.\\
+\underline{Fit Criterion}: System must have access to a mobile camera.\\
+
 \subsection{Implementation Environment of the Current System}
-\lips
+\includegraphics[width=\textwidth]{ImplementationEnvironment.png}
+
 \subsection{Partner or Collaborative Applications}
-\lips
+N/A
 \subsection{Off-the-Shelf Software}
-\lips
+The system requires the following off-the-shelf software:
+\begin{itemize}
+  \item \href{https://github.com/tesseract-ocr/tesseract}{Tesseract} OCR.
+  \item \href{https://flutter.dev/}{Flutter} for Front-end.
+  \item \href{https://nodejs.org/en}{NodeJS} for Back-end.
+  \item \href{https://www.npmjs.com/package/express}{Express} REST API.
+  \item \href{https://www.postgresql.org/}{PostgreSQL} as Relational Database.
+\end{itemize}
+
 \subsection{Anticipated Workplace Environment}
-\lips
+N/A
 \subsection{Schedule Constraints}
-\lips
+The schedule for this project is in adherance with the milestones implemented by our course instructor. Found \href{https://gitlab.cas.mcmaster.ca/courses/capstone/-/blob/main/CourseOutline/Capstone_Outline.pdf}{here}.
 \subsection{Budget Constraints}
-\lips
+No budget is provided for this porject. Any costs that are incurred by the project are at the cost of the developers.
```

Comment:
- You wrote "porject" here rather than "project".
- Also, I think we were allowed a maximum of $600 (even if we're not spending it) if I recall correctly so maybe you can mention that.